components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/util/FrameworkUtils.java

```diff
@@ -3035,4 +3035,17 @@ public static void resetAuthenticationContext(AuthenticationContext context) {
         context.setRetrying(false);
         context.setCurrentAuthenticator(null);
     }
+
+    /**
+     * Check whether the JIT provisioning enhanced feature is enabled.
+     *
+     * @return true if the JIT provisioning enhanced features is enabled else return false.
+     */
+    public static boolean isEnhancedFeature() {
+
+        if (StringUtils.isNotBlank(IdentityUtil.getProperty(FrameworkConstants.ENABLE_ENHANCE_FEATURE))) {
+            return Boolean.parseBoolean(IdentityUtil.getProperty(FrameworkConstants.ENABLE_ENHANCE_FEATURE));
+        }
+        return false;
```

Comment:
Can't we just use `Boolean.parseBoolean`? won't it return `false` when the value if null?

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/3597/commits/6e1589981f43782e7824b22d065010c4ef34b01d